pkg/operator/controllers/genevalogging/genevalogging_test.go

```diff
@@ -154,8 +155,10 @@ func TestGenevaLoggingDaemonset(t *testing.T) {
 			}
```

Comment:
I think we should update these tests to also assert the expected conditions are set for the controller. PRs such as https://github.com/Azure/ARO-RP/pull/2965 have an example of how to use the helper functions defined for this purpose.